src/styles/utils/variables.scss

```diff
@@ -0,0 +1,15 @@
+$color-gray: #eee;
+$color-pink: #ffb6c1;
+$gap: 1px;
+$size-day: 100px;
+$calendar-padding: 10px;
+$month-lengths: 28, 29, 30, 31;
+
+$dayOfWeek:
+  mon 0,
+  tue 1,
+  wed 2,
+  tue 3,
```

Comment:
There is a duplicate 'tue' entry in the $dayOfWeek list. It should be 'thu' instead.